### ISSUE TITLE
[Crash] Cannot play file

### ISSUE THREAD

**Issue by gxgani** [digimezzo/dopamine-windows#78]
```
10:35:45.8119|Error|Dopamine.Common.Services.Playback.PlaybackService+VB$StateMachine_96_TryPlayAsync.MoveNext|Could not play the file X:\Some_Drive\Loui & Scibi, Fourfeet - More Than Enough [Lou Van Remix] [SM].mp3. LoopMode=None, Shuffle=False. Exception: Interops::MFCreateSourceReaderFromURL caused an error: 0xc00d3e8c, "The Media format is recognized but is invalid.".. StackTrace:    at CSCore.MediaFoundation.MediaFoundationCore.CreateSourceReaderFromUrlNative(String url)
   at CSCore.MediaFoundation.MediaFoundationDecoder..ctor(String url)
   at CSCore.Codecs.CodecFactory.GetCodec(String filename)
   at Dopamine.Core.Audio.CSCorePlayer.InitializeSoundOut()
   at Dopamine.Core.Audio.CSCorePlayer.Play(String iFilename)
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Dopamine.Common.Services.Playback.PlaybackService.VB$StateMachine_96_TryPlayAsync.MoveNext()
2016-07-28 10:35:46.0785|Error|Dopamine.Application.ExecuteEmergencyStop|Unhandled Exception. Exception:
System.ArgumentNullException: Value cannot be null.
   at System.Threading.Monitor.Enter(Object obj)
   at CSCore.Win32.ComObject.Finalize()

Stack trace:
   at System.Threading.Monitor.Enter(Object obj)
   at CSCore.Win32.ComObject.Finalize()

2016-07-28 10:35:46.0785|Info|Dopamine.Application.ExecuteEmergencyStop|### FORCED STOP of Dopamine, version 1.0 (Build 660) ###
```

**Comment by digimezzo** [comment]
Could you send me the file that could not be played so I can test it here?

**Comment by gxgani** [comment]
Kindly check your Support Mail.

**Comment by gxgani** [comment]
Hey, any word on the issue?

**Comment by digimezzo** [comment]
I'm working on it. Should be fixed in the next release.

**Comment by digimezzo** [comment]
The bug is fixed in 474aa8371f2ec6b3c813bfe69ffbc6993eab0e72

**Comment by digimezzo** [comment]
This issue needs further invetigation. It seems the MP3 in question is considered invalid by MediaFoundation. Most audio players seem to be able to play it though. The only other player that fails to play it is Microsoft Groove.
